docs/2.7.0/docs/daml-finance/overview/intro.rst

```diff
@@ -74,7 +74,7 @@ be modeled:
 * Cross-entity issuance: atomic, multi-party issuance across investors, issuer, risk book, and
   treasury
 * Asset-agnostic trading facility: generic delivery-vs-payment and immediate, guaranteed settlement
-* Exotic asset types: non-fungible and non-transferable assets
+* Exotic asset types: non-fungible and non-reassignable assets
```

Comment:
This change is not correct. One of the reasons we want to stop using the word "transfer" for changing the mapping between contract and domain is that assets actually _are_ transferred among owners and from one participant node to another.   This paragraph is actually referring to assets and asset transfers, not the reassignment that changes the contract-domain mapping.

Reply:
Thanks, this is exactly what I was looking for.